config/reader.go

```diff
@@ -4,57 +4,69 @@ import (
 	"bufio"
 	"fmt"
 	"os"
+	"path/filepath"
+	"runtime"
 	"strings"
 	"sync"
+
+	"github.com/rancher/distros-test-framework/pkg/logger"
 )
 
 var (
 	product *Product
 	once    sync.Once
+	l       = logger.AddLogger()
```

Comment:
can we rename it to `logger`

Reply:
here we cant, its the pkg name

i can rename to `log`, if thats better